recipes/r-islr2/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export DISABLE_AUTOBREW=1
+
+mv DESCRIPTION DESCRIPTION.old
+grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
I think the `Priority` is not relevant anymore.

Reply:
Do you mean I should remove line 6, or?

Reply:
Both 5 and 6 can be dropped.

Reply:
✅